### PR TITLE
Fix Literal import for Python3.7

### PR DESCRIPTION
**Pull request by henriquesalvaro**
Hey folks, just ran into import issues related to the `Literal` import from `typing` - our project is still in `3.7` version, and [`Literal` was added on version `3.8`](https://github.com/python/typing_extensions).
Not sure if supporting `3.7` is something you had in mind, but here is the "fix" for it, just in case.

Most of the `e2e_tests` ran fine - with exception of the ones that have hardcoded values for `user_id` and details like that - appreciate it if any of you can run them with the "right" token 👀 to confirm.

P.S: `typing_extensions` is probably not needed on any of the `requirements` files, because of the `pytest` requirement already, but figured it wouldn't hurt to be explicit about what I've changed.

### CODE REVIEW

File: unit/models/event.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import json
 from datetime import datetime, date
-from typing import Literal, Optional
+try:
+    from typing import Literal, Optional
+except ImportError:
+    from typing import Optional
```
Comment (AsafPolakovski) [inline unit/models/event.py:6]: @henriquesalvaro dont you want typing_extensions here?

File: unit/models/check_deposit.py [hunk 2]
```diff
@@ -1,6 +1,10 @@
 import json
 from unit.utils import date_utils
-from typing import Optional, IO, Literal
+try:
+    from typing import Optional, IO
```
Comment (AsafPolakovski) [inline unit/models/check_deposit.py:4]: @henriquesalvaro I think you forgot Literal here.
```suggestion
    from typing import Optional, IO, Literal
```

**Comment by AsafPolakovski** [comment]
Hey @henriquesalvaro I left 1 comment, other than that it looks good to me.
Would like to get @axshani opinion as well.

**Comment by AsafPolakovski** [comment]
@henriquesalvaro it seems like the tests are failing now, did you manage to run the tests on your sandbox env?
We will also try to debug them on our side in order to help you get this pr merged :)

**Comment by AsafPolakovski** [comment]
@henriquesalvaro we merged a fix for the tests, can you pull master into your branch and push again so the CI runs?

Thanks :)

**Comment by henriquesalvaro** [comment]
> @henriquesalvaro it seems like the tests are failing now, did you manage to run the tests on your sandbox env? We will also try to debug them on our side in order to help you get this pr merged :)

I had managed to run all the ones that weren't using specific account/customer ids.
I'll look into them, thanks!

**Comment by henriquesalvaro** [comment]
The tests that failed here are related to webhooks weirdly enough 🤔 . They seem fine on my branch (local):
image

**Review by axshani — APPROVED** [review]

**Pull request merged**